src/main/java/org/jitsi/videobridge/util/config/JvbConfig.java

```diff
@@ -20,9 +20,9 @@
 import org.jetbrains.annotations.*;
 import org.jitsi.utils.logging2.*;
 
-import java.nio.file.*;
 import java.util.*;
-import java.util.function.*;
+
+import static org.jitsi.videobridge.util.config.ConfigSupplierSettings.*;
```

Comment:
How messy would it be to not make this `import static`?  I found it sort of confusing how the `ConfigSupplierSettings` were affecting `JvbConfig` until I noticed it, so I'm wondering if it'd be clearer if `JvbConfig` explicitly mentioned `ConfigSupplierSettings` where it was needed.

Reply:
not messy, i can change it